content/scripts/build.js

```diff
@@ -113,18 +110,20 @@ function buildMDNUrl(locale, slug) {
   return `/${locale}/docs/${slug}`;
 }
 
-/** Throw an error if the locale in insane.
- * This helps breaking the build if someone has put in faulty data into
- * the content (metadata file).
- * If all is well, do nothing. Nothing is expected to return.
- */
-function validateLocale(locale) {
+function extractLocale(source, folder) {
+  // E.g. 'pt-br/web/foo'
+  const relativeToSource = path.relative(source.filepath, folder);
+  // E.g. 'pr-br'
+  const localeFolderName = path.parse(relativeToSource).dir.split(path.sep)[0];
```

Comment:
Hm, it seems to me that this could be simplified to:
```suggestion
  const localeFolderName = relativeToSource.split(path.sep)[0];
```

Reply:
done.